.github/workflows/tests.yaml

```diff
@@ -20,9 +20,13 @@ jobs:
       matrix:
         include:
           - php-version: "8.1"
-            dependencies: "lowest"
+            dependencies: "lowest" # Pimcore 10.5.0
           - php-version: "8.1"
-            dependencies: "highest"
+            dependencies: "highest" # Pimcore 11.*
+          - php-version: "8.2"
+            dependencies: "lowest" # Pimcore 10.5.0
```

Comment:
Pimcore 10 isn't compatible with PHP 8.2.

Reply:
Hmm, the Pipeline installs 10.5.0. https://github.com/teamneusta/pimcore-presentation-bundle/actions/runs/6510813888/job/17685275884?pr=30#step:4:177

Maybe because of `^8.0`. Any ideas how to fix that in pipeline?